quickwit/quickwit-search/src/cluster_client.rs

```diff
@@ -157,6 +161,35 @@ impl ClusterClient {
         // TODO: implement retry
         client.leaf_list_terms(request.clone()).await
     }
+
+    /// Attempts to store a given search context within the cluster.
+    ///
+    /// This function may fail silently, if no clients was available.
+    pub async fn put_search_after_context(
+        &self,
+        put_search_after_context_req: PutSearchAfterContextRequest,
+    ) {
+        let clients = self
+            .search_job_placer
+            .best_nodes_per_affinity(&put_search_after_context_req.search_after_id)
+            .await;
+        let mut successful_replication = 0;
```

Comment:
```suggestion
        // TODO: Issue concurrent requests
```

Reply:
We don't have to issue the requests sequentially.

Reply:
ah ok got it!